src/App.js

```diff
@@ -1,28 +1,25 @@
-import React, {Component}  from 'react';
-import logo from './logo.svg';
+import React from 'react';
+import {HashRouter, Route} from "react-router-dom";
 import './App.css';
 import './scss/Main.scss';
+import Start from "./components/Start";
+import City from "./components/City";
+import Block from "./components/Block";
+import blocks from "./data/data";
 
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    return (
+        <HashRouter>
+            <>
+                <Route exact path="/" component={Start}/>
+                <Route exact path="/:city" component={City}/>
+                <Route exact path="/:city/:name" render={props => <Block {...props} blocks={blocks}/>}/>
```

Comment:
I'd suggest something like `path=/:city/area/:name` --> it would be easier to read and maintain if you have "ritcher" URL

Reply:
Even `path=/:city/aree/:name/markets/:id`

Example url URL: `warsaw/aree/ochota/markets/88f7b4ce-6933-11ea-bc55-0242ac130003`